2022-05-17T16:07:18.132, Exception was thrown: The database is not available
se.kth.iv1350.hackers.integration.InventorySystemException: The database is not available
	at se.kth.iv1350.hackers.integration.InventorySystem.requestItemInfo(InventorySystem.java:46)
	at se.kth.iv1350.hackers.integration.DBController.requestItemInfo(DBController.java:38)
	at se.kth.iv1350.hackers.controller.Controller.requestItemInfo(Controller.java:46)
	at se.kth.iv1350.hackers.controller.Controller.addItem(Controller.java:67)
	at se.kth.iv1350.hackers.view.View.sampleExecution(View.java:56)
	at se.kth.iv1350.hackers.startup.Main.main(Main.java:26)
2022-05-17T16:07:18.140, Exception was thrown: The database is not available
se.kth.iv1350.hackers.integration.InventorySystemException: The database is not available
	at se.kth.iv1350.hackers.integration.InventorySystem.requestItemInfo(InventorySystem.java:46)
	at se.kth.iv1350.hackers.integration.DBController.requestItemInfo(DBController.java:38)
	at se.kth.iv1350.hackers.controller.Controller.requestItemInfo(Controller.java:46)
	at se.kth.iv1350.hackers.controller.Controller.addItem(Controller.java:67)
	at se.kth.iv1350.hackers.view.View.sampleExecution(View.java:62)
	at se.kth.iv1350.hackers.startup.Main.main(Main.java:26)
